src/OrchardCore.Modules/OrchardCore.Media/Services/IUserAssetFolderNameProvider.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OrchardCore.Media.Services
+{
+    public interface IUserAssetFolderNameProvider
```

Comment:
Can you move this interface to `OrchardCore.Media.Abstractions` (in the root folder, with `OrchardCore.Media` as the namespace please.

Reply:
done